play/src/front/Phaser/Entity/Character.ts

```diff
@@ -336,7 +336,7 @@ export abstract class Character extends Container implements OutlineableInterfac
         }
         this.playAnimation(this.lastDirection, true);
 
-        this.setDepth(this.y);
+        this.setDepth(this.y + 16);
```

Comment:
By default origin is set as center (0.5) with 32px of height we need to add 16px to the depth level so it can be counted as "feet first"
![obraz](https://user-images.githubusercontent.com/14911927/212890372-772aad62-8072-4dab-98f5-accf72ffb1e9.png)